README.md

```diff
@@ -98,7 +95,10 @@ To run different web applications for diffent virtual hosts:
                                   (e.g., xls=application/vnd.ms-excel:wmf=application/x-msmetafile)
        --maxParamCount=N        = set the max number of parameters allowed in a form submission to protect
                                   against hash DoS attack (oCERT #2011-003). Default is 10000.
-       --useJmx                 = Enable Jetty JMX                                   
+       --useJmx                 = Enable Jetty JMX    
+       --qtpMaxThreads          = max threads number when using Jetty Queued Thread Pool                           
```

Comment:
Maybe it makes sense to explicitly add "Number" to the argument name. Does not matter much tho

Reply:
I think we generally seem to be using "count" - concur that it might make sense to include that, i.e. "qtpMaxThreadCount" and "jettyAcceptorCount" and "jettySelectorCount", etc

But that's more of a style point -- up to the PR author if they want to or not (it's hardly a PR blocker).

Reply:
@svanoort I like this naming proposal! Style it's very important! :)